### PR TITLE
Add `CODE_OF_CONDUCT.md`

### PR DESCRIPTION
**Pull request by mertyildiran**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
```
Comment (IgorGov) [inline CODE_OF_CONDUCT.md:1]: Can we add it to the .github folder?

Comment (mertyildiran) [reply CODE_OF_CONDUCT.md:1]: @IgorGov the right place for this file is the root directory as far as I know. See the GitHub's related page https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-code-of-conduct-to-your-project

Comment (IgorGov) [reply CODE_OF_CONDUCT.md:1]: Github suggest 3 places root, docs, .github. I prefer to have one place for all github related docs, and prefer it not to be the root as it has many unrelated files.

From your link: 
To make your code of conduct visible in the repository's root directory, type CODE_OF_CONDUCT in the file name field.
To make your code of conduct visible in the repository's docs directory, type docs/CODE_OF_CONDUCT.
To make your code of conduct visible in the repository's .github directory, type .github/CODE_OF_CONDUCT.

Comment (mertyildiran) [reply CODE_OF_CONDUCT.md]: @IgorGov oh yes correct, I'm sorry. I'm just too used to seeing it almost always in the root directory but OK. Moved it to `.github/` directory.

**Review by IgorGov — APPROVED** [review]

**Pull request merged**